orchestra/todos/views.py

```diff
@@ -37,13 +41,41 @@ def update_todos_from_todolist_template(request):
         raise BadRequest('TodoList Template not found for the given slug.')
 
 
+@api_endpoint(methods=['GET'],
+              permissions=(permissions.IsAuthenticated,
+                           IsAssociatedWithProject),
+              logger=logger)
+def worker_recent_todo_qas(request):
+    """
+    The function returns TodoQAs from the requesting user's most recent
+    task assignment provided they were created in the last 90 days.
+    """
+    try:
+        todoQAs = TodoQA.objects.filter(
```

Comment:
i feel like for this description clobbering to work, you have to order by (is this the same task, older tasks)

Reply:
Fixed it and updated the tests to make sure that we are only loading todo qas from the most recent task for the given worker.